### ISSUE TITLE
Movies from Radarr lists are added to root.

### ISSUE THREAD

**Issue by darzur** [Radarr/Radarr#3036]
It's not possible to define target folder when creating Radarr list. Even if selected it always returns to blank after click "Save". As a result all movies fetched from Radarr lists are added to root of my device (RPi)

1. Go to Settings -> Lists
2. Click on big "+" and select Radarr Lists
3. Fill all fields and select destination folder from dropdown list
4. Click "Save"
5. Folder field will be blanked upon save.
6. All movie folders from that particular list will be created in root

**Expected behavior**
Movie folders will be created under selected patch.

**Desktop (please complete the following information):**
 - OS: Raspbian Version: 0.2.0.1120 Mono: 4.2.3 (Stable 4.2.3.4/832de4b Wed Mar 16 13:34:50 UTC 2016)

![radarr](https://user-images.githubusercontent.com/5848983/45636526-3e761c00-baa8-11e8-9500-7a0d1a31a3f6.png)

**Comment by chrisfoster1971** [comment]
I have a similar issue whereby movies added via the watch list are defaulting to /moviename (year) not /vokume/media/Movies set as default location however from within Radarr are added correctly. Running v0.2.0.1191 on osx

**Comment by fryfrog** [comment]
This is a long standing issue, I think. You should be able to work around it by selecting a *different* path, saving it and then going back and selecting the *right* path and saving it. Never go back to it again, but if you do... repeat the above. :/

**Comment by darzur** [comment]
That worked, thanks. :) It's not a solution however so I will not mark it as closed.

**Comment by fryfrog** [comment]
Yeah, and one that I swear is fixed... but clearly isn't.

@galli-leo, this needs a look. :)

**Comment by Dulanic** [comment]
I dont think this is only firefox. Mine adds to root and I use Chrome. My lists have a folder selected but it does not put it there.

**Comment by rfgamaral** [comment]
I've been experience similar issues but after some testing this is what I observed on my end:

- The select box is working as expected in both Firefox/Chrome and it saves the proper path you select, although it doesn't seem to save the path (but it is) because:
  - Firefox: Reopening the list doesn't show the path as selected, the select box is empty. But like I said, I tried to import the movies from the list and they were imported to the correct folder.
  - Chrome: Reopening the list does show the previously selected path. This rendering issue seems to be Firefox only.
- To me, this looks like it's happening only if the folder selected is the first one in the list. If you pick the second folder (or any other after), the issue above shouldn't occur in both Firefox/Chrome.
- Saving a list turns the select box empty right before the modal dialog is closed but this is just a visual glitch, the folder is still saved correctly to the database. Reopening it should show the correct folder (at least on Chrome).
- I have tested this only with Trakt.tv lists, not with "Radarr Lists", but it should't make a difference IMO.

I do have a PR incoming to change a bit of code on the lists part, it might make things more consistent and maybe fix this issue. However, I can't be 100% sure until you guys test it for yourselves because I'm not observing the main issue as some of you.

The only thing I wasn't able to fix is the visual glitch where the select box becomes empty while saving, can't understand what's happening here, everything I tried didn't work (this happens in both Firefox/Chrome/Edge - not sure about Safari). But this is just a visual glitch, path is properly saved.
